Cargo.toml

```diff
@@ -6,21 +6,22 @@ readme = "README.md"
 license = "MIT"
 edition = "2021"
 
+[features]
+default = ["static-model", "auto_resample"]
+static-model = []
+auto_resample = ["dep:rubato"]
+serde = ["dep:serde"]
+
 [dependencies]
 anyhow = "1.0.86"
+hound = "3.5.1"
```

Comment:
I wouldn't add any audio file processing to this project. Have the users do that themselves